independent-publisher-connectors/Mockaroo/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Mockaroo API Key",
+          "description": "The Mockaroo API key which can be found in the My Account area at www.mockaroo.com",
+          "tooltip": "Provide the Mockaroo API key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@rwilson504 iconBrandColor for independent publisher connectors should be `#da3b01`, please change it to this value.